### ISSUE TITLE
netrocks/sftp: copy/move behavior with masks

### ISSUE THREAD

**Issue by unxed** [elfmz/far2l#1167]
1. Connect to SFTP host in netrock
2. Select any file
3. Copy it to *.bak
4. Folder `*.bak` will be created on other panel, and file will be downloaded into it

Expected behavior: file should be copied on the host side to the new file with same name and extension `.bak`.

Same with file move.

**Comment by elfmz** [comment]
yep, its not implemented now for copy/move, but, FYI rename should handle masks. Rename is done by F6 -> remove path/ from path/name in edit
